### PR TITLE
Ria 3278 state payment pending

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3278

### Change description ###
RIA-3278_State_Payment_Pending

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by bradsorour — COMMENTED** [review]
I would consider writing a test for the AC "the appeal can be seen if in the dashboard we search by state Payment pending". Something similar to this: https://github.com/hmcts/ia-ccd-e2e-tests/pull/268/commits/0fb9414aa0624a9057f2d7c87013b65efd7ec12c#diff-2fa9d6924f2a09d6bc5c00f951c6367c

**Comment by Lee365** [comment]
> I would consider writing a test for the AC "the appeal can be seen if in the dashboard we search by state Payment pending". Something similar to this: [0fb9414#diff-2fa9d6924f2a09d6bc5c00f951c6367c](https://github.com/hmcts/ia-ccd-e2e-tests/commit/0fb9414aa0624a9057f2d7c87013b65efd7ec12c#diff-2fa9d6924f2a09d6bc5c00f951c6367c)

Added new test to cover dashboard check for new state of Pending Payment

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Review by mkayad — DISMISSED** [review]
LGTM

File: e2e/features/payment-pending-state-visible-to-tribunal.feature [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending payment state visible for Admin Officer
+
+    Given I am signed in as a `Admin Officer`
+    When I go to the `Case List`
+    Then I should see the `State` field
+    And I should see the option `Pending payment` for the `State` field
+
+  @payment-pending-state-visibility @RIA-3278
+  Scenario: Pending payment state visible for Admin Officer
```
Comment (mkayad) [inline e2e/features/payment-pending-state-visible-to-tribunal.feature:28]: I think it should says Judge here, if not it is repetition of line 20

Comment (Lee365) [reply e2e/features/payment-pending-state-visible-to-tribunal.feature]: Changes name to Judge

**Review by mkayad — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**